phpstan.neon

```diff
@@ -16,7 +17,6 @@ parameters:
     stubFiles:
         - ../../stubs/glpi_constants.php
     ignoreErrors:
-        - '#Call to deprecated method query\(\) of class DBmysql:#'
         - '#Path in include_once\(\) ".*?/files/_plugins/order/generate/custom\.php" is not a file or it does not exist\.#'
     excludePaths:
         - 'inc/referenceinjection.class.php'
```

Comment:
In the code, you need to test for the existence of the class before calling it. That should satisfy phpstan.
```suggestion
```